### ISSUE TITLE
Add support for PDF/A-3a

### ISSUE THREAD

**Issue by fraschfn** [Mc-Zen/zero#115]
when compiling I get this error:

error: PDF/A-3a error: missing alt text
   ┌─ @preview/zero:0.6.1/src/formatting.typ:14:2
   │
14 │   math.equation(sequence(items))
   │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   │
   = hint: make sure your images and equations have alt text

I didn't manage to convert the items parameter of the equation-from-items function into something I could pass to the alt parameter of the equation function, but this workaround did at least suffice to allow the compilation the pdf/a-2a to succeed:

/// Creates an equation from an array of content items. This function leaves
/// the `block` attribute unset.
#let equation-from-items(
  /// An array of content items.
  /// -> array
  items,
) = {
  math.equation(sequence(items), alt: "sorry no equation")
}

**Comment by Mc-Zen** [comment]
Hi @fraschfn ,

The version 0.7.0 of zero introduced built-in support for automatic alt description generation for accessibility!

**Comment by fraschfn** [comment]
Thanks for the information.

I'm using zero via lilaq version 0.6 which still uses zero 0.6.1
I just discovered that you upgraded the version on lilaq already (makes sense if you're the owner of both packages).
So I  just have to wait for a new release of lilaq which is no problem.
